src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.e2e.ts

```diff
@@ -53,6 +68,20 @@ describe('sbb-tooltip-trigger', () => {
     expect(dialog).toHaveAttribute('open');
   });
 
+  it("doesn't show tooltip on keyboard event when disabled", async () => {
+    const tooltipTrigger = await page.find('sbb-tooltip-trigger >>> button');
+    const dialog = await page.find('sbb-tooltip >>> dialog');
+    element.setAttribute('disabled', true);
+
+    await tooltipTrigger.focus();
```

Comment:
I'm not sure how realistic this test ist because you can't focus a disabled button (?).

Reply:
should I remove it or add a spy on the focus event?